### ISSUE TITLE
Suggestion: Add to docs - Can you use JFrog Artifactory or Sonatype Nexus (PyPI mirrors) with Conda-store?

### ISSUE THREAD

**Issue by Adam-D-Lewis** [nebari-dev/nebari-docs#276]
### Context

Users asked this question recently while doing a Nebari install.  Might be useful to have in the docs somewhere.

### Value and/or benefit

Increased understanding

**Comment by pavithraes** [comment]
> Should this be documented on the nebari docs? This seems better suited in the conda-store docs

Yes, we should totally document this in the conda-store docs as well. Since Nebari is one of the main users of conda-store, It think documenting it with Nebari will keep this information in a more searchable/findable place

**Comment by Adam-D-Lewis** [comment]
I believe this is how you could do this, but haven't tried it yet.  Add the following section in the nebari-config.yaml flie  conda_store.extra_settings.CondaStore.pypi_required_packages: ['--index-url=...']
